### PR TITLE
7.0 travel acl

### PR DESCRIPTION
**Pull request by joaoalf**
[IMP] Improved ACL …
- New group Accountant for cost management
- New group Superuser to take the role of other users
- Default ACLs by group
- State change security

### CODE REVIEW

**Comment by pedrobaeza** [comment]
:+1: if Travis is happy.

File: travel/i18n/fr.po [hunk 1]
```diff
@@ -48,30 +48,26 @@ msgstr "Confirmer"
...
 msgstr "Déplacement"
 
 #. module: travel
-#: model:ir.model,name:travel.model_travel_config_settings
-msgid "travel.config.settings"
-msgstr "travel.config.settings"
+#: code:addons/travel/travel.py:146
+#, python-format
+msgid "Only draft travels can be unlinked"
+msgstr "Seulement les déplacements en mode brouillon peuvent être supprimés"
```
Comment (bwrsandman) [inline travel/i18n/fr.po]: Sounds odd.
Try: Seuls les déplacements en mode brouillon peuvent être supprimés
http://www.etudes-litteraires.com/forum/topic20847-accord-de-seul.html

**Comment by bwrsandman** [comment]
1 problem with the translation, but the rest seems good

File: travel/i18n/fr.po (hunk 1 above)
Comment (joaoalf) [inline travel/i18n/fr.po]: @bwrsandman Fixed! Thanks!

**Comment by bwrsandman** [comment]
Waiting for #16 to pass tests

**Comment by joaoalf** [comment]
@bwrsandman The #16 is not a requirement for this one.

**Comment by bwrsandman** [comment]
:+1:

**Pull request merged**